docs: ai updates (#74388)
metabase/metabase: docs/ai/settings.md

# AI settings

> > AI features are available on [Metabase Cloud](https://www.metabase.com/features/metabot-ai) and on self-hosted Metabase, using either the Metabase AI service or your own AI provider API key.

Author: No longer a necessary shoutout

1. Go to **Admin settings > AI**.
2. In **Connect to an AI provider**, choose a **Provider**:
   - **Metabase**: The Metabase AI service. Metabase picks a benchmarked, cost-effective model for you, and charges you on token usage. See [Choose AI provider](#choose-ai-provider)
   - Another supported provider. See [bring your own API key](#bring-your-own-api-key).
3. Once connected, configure [Metabot](#configure-metabot) and other AI features below.

> The Metabase AI add-on only appears in your Metabase Store account after you've connected to the Metabase AI service in **Admin settings > AI**. If you're on a Pro trial and don't see the add-on in **Manage plan**, connect it from Admin first; the Store will reflect it after.

## Choose AI provider

_Admin > AI_

You can choose which AI provider and model is used to power Metabase's built-in agent.

- If you're **self-hosting Metabase** and want to use Metabot, you need to [bring your own AI API key](#bring-your-own-api-key).

> - On **Metabase Cloud**, you can either [bring your own AI API key](#bring-your-own-api-key) or [use Metabase AI provider](#metabase-ai-provider) (comes with an additional charge).

Reviewer: ```suggestion
- On **Metabase Cloud**, you can either [bring your own AI API key](#bring-your-own-api-key) or [use the Metabase AI Service](#metabase-ai-provider).
```

> The AI provider that you specify in AI setting is used only for the Metabase's built-in AI functionality, not the MCP server. See [With the MCP server, your client provides the AI](mcp.md#with-the-mcp-server-your-client-provides-the-ai).

Reviewer: ```suggestion
The AI provider that you specify in AI settings powers Metabase's built-in AI functionality, not the MCP server. [With the MCP server, your client provides the AI](mcp.md#with-the-mcp-server-your-client-provides-the-ai).
```

> ### Metabase AI provider

Author: I really want to have a separate section for anything that charges you money

Reviewer: It's the Metabase AI Service, right?

> ### Metabase AI provider

Author: ```suggestion
### Metabase AI Service
```

> On Metabase Cloud, you can choose to use a Metabase-provided model to power Metabot. This is a good option if you don't have a preferred AI provider, or if you want to manage all AI costs through Metabase only. Metabase selects models automatically. We use internal benchmarks to determine which AI models work best for different tasks, and we're constantly iterating to improve performance.

Reviewer: ```suggestion
On Metabase Cloud, you can have us manage the AI for you with our AI Service.

Metabase's AI Service is a good option if you don't have a preferred AI provider, or if you want to manage all your Metabase AI costs through Metabase. We (Metabase the company) select the models for you. We use internal benchmarks to determine which AI models work best for different tasks, and we're constantly iterating to improve performance.
```

> If you use Metabase AI provider, you'll get charge based on token usage (in addition to your regular Metabase Cloud subscription fee). See [Pricing](https://www.metabase.com/pricing).

Reviewer: ```suggestion
If you use Metabase's AI Service, you'll get charge based on token usage (in addition to your regular Metabase Cloud subscription fee). See [Pricing](https://www.metabase.com/pricing).
```

> To enable Metabase AI provider on Metabase Cloud, you must me logged in to your Metabase instance with an email that matches the email for the admin of your [Metabase Store account](https://store.metabase.com).

Reviewer: ```suggestion
To enable Metabase's AI Service on Metabase Cloud, you must me logged in to your Metabase instance with the email that matches the email for the admin of your [Metabase Store account](https://store.metabase.com).
```

To use Metabase AI provider for your Metabot:

1. Go to **Admin > AI > AI settings**.
2. In **Connect to an AI provider**, choose **Metabase** as the provider.
3. Agree to the terms of service.
4. Click **Connect**.

To disable Metabase AI provider and stop charges:

1. Go to **Admin > AI > AI settings**.
2. Under **Connected to Metabase**, click **Disconnect**.

> Any instance admin - not just people with emails matching a Metabase Store admins - can disconnect Metabase AI provider.

Reviewer: ```suggestion
Any Metabase instance admin can disconnect the Metabase AI Service, even if they lack an admin account in the Metabase store.
```

### Bring your own API key

You can specify your own API key and model for Metabot from one of the supported providers. Currently, Metabase only supports models from Anthropic.

> > Want more AI providers? Submit a [feature request](../troubleshooting-guide/requesting-new-features.md).

Reviewer: ```suggestion
If you're interested in Metabase supporting more AI providers, let us know by submitting a [feature request](../troubleshooting-guide/requesting-new-features.md).
```
Phrasing is longer, but friendlier.

4. Click **Connect**.
5. Select a **Model** from the dropdown. Available models are fetched from the provider using your API key.

> When your connection is active, the provider card header shows **Connected to [provider]** (for example, "Connected to Anthropic") next to a green status dot. With your key connected, you get access to [Metabot](./metabot.md), and [inline SQL generation](./metabot.md#inline-sql-editing).

Author: MCP does not need an AI provider

Click **Regenerate suggested prompts** to generate a fresh set of prompts. You can also run individual prompts to test Metabot's answers, or delete prompts that aren't useful. The Internal and Embedded tabs each maintain their own set of suggestions, so regenerating on one tab doesn't affect the other.

> ## Enable MCP server

Author: Moved to MCP doc
